lib/parity/backup.rb

```diff
@@ -31,6 +31,15 @@ def restore_from_development
     end
 
     def restore_to_development
+      drop_development_database
+      pull_remote_database_to_development
+    end
+
+    def drop_development_database
+      Kernel.system("dropdb #{development_db}")
```

Comment:
http://www.postgresql.org/docs/9.2/static/app-dropdb.html

`dropdb` is a Postgres-installed executable that drops a given database. It's much cleaner than using `psql`, because we don't have to assume the user's maintenance database is named `postgresql` or use credentials and hostnames, etc.